+++ a/arch/arm/mach-kirkwood/sheevaplug-setup.c
+++ b/arch/arm/mach-kirkwood/sheevaplug-setup.c
@@ -62,11 +62,17 @@
 
 static struct gpio_led sheevaplug_led_pins[] = {
 	{
-		.name			= "plug:green:health",
-		.default_trigger	= "default-on",
-		.gpio			= 49,
+		.name			= "dockstar:green:health",
+		.default_trigger	= "none",
+		.gpio			= 46,
 		.active_low		= 1,
 	},
+  {
+		.name			= "dockstar:orange:health",
+		.default_trigger	= "default-on",
+		.gpio			= 47,
+		.active_low		= 1,
+  }
 };
 
 static struct gpio_led_platform_data sheevaplug_led_data = {
@@ -84,7 +90,8 @@
 
 static unsigned int sheevaplug_mpp_config[] __initdata = {
 	MPP29_GPIO,	/* USB Power Enable */
-	MPP49_GPIO,	/* LED */
+	MPP46_GPIO,	/* LED green */
+	MPP47_GPIO,	/* LED orange */
 	0
 };
 
